### PR TITLE
.rmd file for slidy added

### PR DESCRIPTION
**Pull request by Jennieus**

### CODE REVIEW

File: rmarkdown/The Katak Detector.Rmd [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+b.    Monitoring indicator species can also better help scientists their research on ecology and biodiversity. For example, while a wetland ecosystem may show general signs that something is amiss, if scientists see that frog populations are plummeting, they’ll know the problem may be pesticides.
+c.    Scientists, lawmakers, and public officials can also use the data collected from indicator species to implement conservation policies. For examples, protecting land and habitats, regulating development or chemical use, or passing laws to reverse or prevent damage before it becomes too late.
+
+- Project challenges & limitations
+
+Shinyapp allocates only maximum size of 1 GB for the Free and Starter plans
+
+i.    Reduce the data up to 2020 since it can't handle more than 44k rows of data (Original data has 193792k rows)
+ii.   Shinyapps keeps on crashing & disconnected from server due to too many data & memory usage of libraries used
+iii.    Due to computation limits, we omitted the usage of mapview library as it consumed too much memory
```
Comment (yuenherny) [inline rmarkdown/The Katak Detector.Rmd]: It's actually Leaflet library

File: rmarkdown/The Katak Detector.Rmd [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+>https://challenge.ey.com/challenges/level-2-global-frog-discovery-tool-EEmnW-5fR/data-description
+
+
+## Introduction of Project
+
+- Problem statements
+
+The declination of frogs may be a warning to us that our planet is becoming unliveable. The disappearance of frogs is causing concerns such as increase of greenhouse effect, climate change & balance of ecosystem.
+
+- Research questions
```
Comment (yuenherny) [inline rmarkdown/The Katak Detector.Rmd]: We didn't really answered these questions. Perhaps consider dropping those unanswered?

File: rmarkdown/The Katak Detector.Rmd [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+Shinyapp allocates only maximum size of 1 GB for the Free and Starter plans
+
+i.    Reduce the data up to 2020 since it can't handle more than 44k rows of data (Original data has 193792k rows)
+ii.   Shinyapps keeps on crashing & disconnected from server due to too many data & memory usage of libraries used
+iii.    Due to computation limits, we omitted the usage of mapview library as it consumed too much memory
+iv.   Unable to showcase prediction outcome, since we dropped huge chunk of data
+
+- Future recommendations
+
+Build a computational model that can predict the count of frogs for a specific location using multiple datasets, and to validate model on other locations of other countries using a variety of open-source geospatial datasets available on Microsoft Planetary Portal (TerraClimate, Sentinel L-2, Copernicus Elevation, Esri 10m Land Cover and JRC Water Surface).
```
Comment (yuenherny) [inline rmarkdown/The Katak Detector.Rmd]: Maybe we can add another recommendation:
- Host the data in a separate database to reduce Shiny server load.

**Review by yuenherny — CHANGES_REQUESTED** [review]
Just minor edits. Should be suitable to merge after the edit.

File: rmarkdown/The Katak Detector.Rmd [hunk 4]
```diff
@@ -32,8 +32,7 @@ The declination of frogs may be a warning to us that our planet is becoming unli
 
 i.	    What are the frog trends that we can see from the dataset?
 ii.	    When the frog’s population at its highest?
-ii.	    When and where the frog’s population at its concerned rates?
-iv.   	Which species is considered dominant population in Australia?
+ii.	    Which species is considered dominant population in Australia?
```
Comment (yuenherny) [inline rmarkdown/The Katak Detector.Rmd]: Is it `iii`?

**Review by yuenherny — CHANGES_REQUESTED** [review]

File: rmarkdown/The Katak Detector.Rmd (hunk 4 above)
Comment (Jennieus) [reply rmarkdown/The Katak Detector.Rmd]: ohhh typo

**Review by yuenherny — APPROVED** [review]
Looks good.

**Pull request merged**